packages/collector/test/immediate/package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "immediate",
+  "version": "1.0.0",
+  "description": "test application",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/instana/nodejs-sensor.git"
+  },
+  "main": "../apps/express",
```

Comment:
TODO: This doesn't make sense yet. We need an uninstrumented app here, and ../apps/express is instrumented manually.